source/blog/2017-03-30-minishift-intro.html.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Introducing Minishift - Run OpenShift locally
+date: 2017-03-30 05:45 UTC
+author: budhram
+published: true
+comments: true
+tags: minishift, atomic-developer-bundle, openshift-local
+---
+
+It has been quite a few months since our [last blog post](http://www.projectatomic.io/blog/2016/08/vagrant-service-manager-1-3-0-release/) on [vagrant-service-manager](https://github.com/projectatomic/vagrant-service-manager) as we have been looking around for alternative solutions to replace the [Atomic Developer Bundle (ADB)](https://github.com/projectatomic/adb-atomic-developer-bundle). A separate blog post will address the reasons for this.
+
+With this blog post, we introduce [Minishift](https://github.com/minishift/minishift), an alternative to ADB, which is a tool that helps you run OpenShift locally by running a single-node OpenShift cluster inside a VM. You can try out OpenShift or develop with it, day-to-day, on your local host.
```

Comment:
We should remove `an alternative to ADB` as we mentioned above that we moved to Minishift as it provides better user experience.